{
  "timestamp": "20250831_103210",
  "dockerfile_path": "envgym/envgym.dockerfile",
  "image_name": "envgym_test_1756636329",
  "build": {
    "success": false,
    "stdout": "",
    "stderr": "#0 building with \"default\" instance using docker driver\n\n#1 [internal] load build definition from envgym.dockerfile\n#1 transferring dockerfile: 1.37kB done\n#1 DONE 0.0s\n\n#2 resolve image config for docker-image://docker.io/docker/dockerfile:1.4\n#2 DONE 0.2s\n\n#3 docker-image://docker.io/docker/dockerfile:1.4@sha256:9ba7531bd80fb0a858632727cf7a112fbfd19b17e94c4e84ced81e24ef1a0dbc\n#3 CACHED\n\n#4 [internal] load .dockerignore\n#4 transferring context: 2B done\n#4 DONE 0.0s\n\n#5 [internal] load metadata for docker.io/continuumio/miniconda3:4.7.12\n#5 DONE 0.1s\n\n#6 [stage-0  1/11] FROM docker.io/continuumio/miniconda3:4.7.12@sha256:6c979670684d970f8ba934bf9b7bf42e77c30a22eb96af1f30a039b484719159\n#6 DONE 0.0s\n\n#7 [internal] load build context\n#7 transferring context: 311B done\n#7 DONE 0.0s\n\n#8 [stage-0  2/11] WORKDIR /app\n#8 CACHED\n\n#9 [stage-0  6/11] RUN conda create -y -n ssense python=3.7 &&     echo \"conda activate ssense\" >> ~/.bashrc\n#9 CACHED\n\n#10 [stage-0  7/11] COPY requirements.txt .\n#10 CACHED\n\n#11 [stage-0  3/11] RUN echo \"deb http://archive.debian.org/debian buster main\" > /etc/apt/sources.list     && echo \"deb http://archive.debian.org/debian-security buster/updates main\" >> /etc/apt/sources.list\n#11 CACHED\n\n#12 [stage-0  4/11] RUN --mount=type=cache,target=/var/cache/apt     apt-get update &&     apt-get install -y wget unzip &&     rm -rf /var/lib/apt/lists/*\n#12 CACHED\n\n#13 [stage-0  5/11] RUN mkdir -p plots models results\n#13 CACHED\n\n#14 [stage-0  8/11] RUN --mount=type=cache,target=/root/.cache/pip     pip install -r requirements.txt\n#14 CACHED\n\n#15 [stage-0  9/11] RUN wget -O dataset.zip \"https://zenodo.org/record/6388301/files/sixthsense-dataset-v2.0.zip?download=1\" &&     unzip dataset.zip -d csvs/ &&     rm dataset.zip\n#15 0.224 --2025-08-31 10:32:10--  https://zenodo.org/record/6388301/files/sixthsense-dataset-v2.0.zip?download=1\n#15 0.232 Resolving zenodo.org (zenodo.org)... 188.185.45.92, 188.185.48.194, 188.185.43.25, ...\n#15 0.355 Connecting to zenodo.org (zenodo.org)|188.185.45.92|:443... connected.\n#15 0.678 HTTP request sent, awaiting response... 301 MOVED PERMANENTLY\n#15 0.813 Location: /records/6388301/files/sixthsense-dataset-v2.0.zip [following]\n#15 0.813 --2025-08-31 10:32:10--  https://zenodo.org/records/6388301/files/sixthsense-dataset-v2.0.zip\n#15 0.813 Reusing existing connection to zenodo.org:443.\n#15 0.813 HTTP request sent, awaiting response... 404 NOT FOUND\n#15 0.958 2025-08-31 10:32:10 ERROR 404: NOT FOUND.\n#15 0.958 \n#15 ERROR: process \"/bin/sh -c wget -O dataset.zip \\\"https://zenodo.org/record/6388301/files/sixthsense-dataset-v2.0.zip?download=1\\\" &&     unzip dataset.zip -d csvs/ &&     rm dataset.zip\" did not complete successfully: exit code: 8\n------\n > [stage-0  9/11] RUN wget -O dataset.zip \"https://zenodo.org/record/6388301/files/sixthsense-dataset-v2.0.zip?download=1\" &&     unzip dataset.zip -d csvs/ &&     rm dataset.zip:\n0.224 --2025-08-31 10:32:10--  https://zenodo.org/record/6388301/files/sixthsense-dataset-v2.0.zip?download=1\nconnected.\n301 MOVED PERMANENTLY\n0.813 Location: /records/6388301/files/sixthsense-dataset-v2.0.zip [following]\n0.813 --2025-08-31 10:32:10--  https://zenodo.org/records/6388301/files/sixthsense-dataset-v2.0.zip\n0.813 Reusing existing connection to zenodo.org:443.\n0.813 HTTP request sent, awaiting response... 404 NOT FOUND\n0.958 2025-08-31 10:32:10 ERROR 404: NOT FOUND.\n0.958 \n------\nERROR: failed to solve: process \"/bin/sh -c wget -O dataset.zip \\\"https://zenodo.org/record/6388301/files/sixthsense-dataset-v2.0.zip?download=1\\\" &&     unzip dataset.zip -d csvs/ &&     rm dataset.zip\" did not complete successfully: exit code: 8\n"
  },
  "run": {
    "success": false,
    "stdout": "",
    "stderr": ""
  }
}